relay-server/src/services/processor/replay.rs

```diff
@@ -88,6 +90,8 @@ pub fn process(
                 item.set_payload(ContentType::OctetStream, replay_recording);
             }
             ItemType::ReplayVideo => {
+                contains_mobile_event = true;
```

Comment:
I assume `Video == Mobile`, but can't we name the feature that way too, we just have a feature that toggles the videos? Just from reading the feature I wouldn't have expected this.

Reply:
Renamed to "video".